.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,95 @@
+## Preview link
+Preview link:
+<!-- If available, provide a link to a demo of the solution in action. -->
+
+## Problem statement
+_Summarize the problem this PR solves in a clear and concise statement._
+<!--
+A successful problem statement conveys:
+1. The desired state,
+2. The actual state, and
+3. Consequences of remaining in the current state (who does this affect and to what degree?)
+-->
 
-## Additional information
+## Solution
+_Provide a summary of the solution this PR offers._
+<!--
+It can be helpful if we understand:
+1. What the solution is,
+2. Why this approach was chosen,
+3. How you implemented the change, and
+4. Possible limitations of this approach and alternate solution paths.
+-->
+
+## Major changes
+_For complex PRs, create a list of the significant updates made._
+
+## Testing and review
+_Share recommended methods for reviewing this change._
+<!--
+1. Describe the tests that you ran to verify your changes,
+2. Provide instructions to reproduce these tests, and
+3. Clarify the type of feedback you are looking for at this phase.
+-->
 
-Include any of the following (as necessary): 
+## Dependency updates
+_For PRs that include dependency updates, include a list of the changed dependencies and version numbers._
+| Dependency name              | Previous version | New version |
+| ---------------------------- | :--------------: | :---------: |
+| [Updated dependency example] |     [1.0.0]      |   [1.0.1]   |
+| [New dependency example]     |        --        |   [3.0.1]   |
+| [Removed dependency example] |     [2.10.2]     |     --      |
 
-* Relevant research and support documents
-* Type of content review needed: stylistic or copy editing
-* Screenshot images
-* Notes
+---
 
-Before you hit Submit, make sure you’ve done whichever of these applies to you:
+Before opening this PR, make sure you’ve done whichever of these applies to you:
 
-- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
-- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
+- [ ] Confirm that this code follows the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
+- [ ] Run `git pull origin [base branch]` to pull in the most recent updates from your base and check for merge conflicts. (Often, the base branch is `develop`).
+- [ ] Run `npm run prettier:scss` to format any Sass updates.
+- [ ] Run `npm test` and confirm that all tests pass.
 - [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
-- [ ] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.
```

Comment:
Possibly convert these checkboxes to comments/remove

Reply:
Commented this out in [e232d26](https://github.com/uswds/uswds/pull/4842/commits/e232d26553280d8f1f936476409f95c6c492bd29)